nym-vpn-apple/Services/Sources/Services/ConfigurationManager/ConfigurationManager.swift

```diff
@@ -1,13 +1,21 @@
 import Foundation
+import AppSettings
 import Constants
 import Logging
 
 public final class ConfigurationManager {
+    private let appSettings: AppSettings
     private let logger = Logger(label: "Configuration Manager")
 
-    private var currentEnv: Env = .mainnet
+    // Source of truth in AppSettings.
+    // We need to set same settings in tunnel extension as well.
+    private var currentEnv: Env = .mainnet {
```

Comment:
Maybe then you should simply define a getter and setter that both proxy to `appSettings.currentEnv`?